avrohugger-tools/src/test/scala/Util.scala

```diff
@@ -1,15 +1,15 @@
 
-import java.io.BufferedReader
-import java.io.File
-import java.io.FileReader
-import java.io.IOException
+import org.specs2.matcher.{ Expectable, ExpectationsCreation, MatchResult, Matcher }
+import org.specs2.matcher.Matchers.typedEqualExpectation
 
 object Util {
   
   def readFile(fileName: String, maxTries: Int = 3): String = {
     def readFile0(count: Int): String = {
       try { // if file is empty, try again, it should be there
-        val contents: String = scala.io.Source.fromFile(fileName).mkString
+        val source = scala.io.Source.fromFile(fileName)
+        val contents: String = source.mkString
+        source.close()
```

Comment:
stream wasn't closed and in this a rec function, which breaks on the 2nd execution